src/site/_includes/partials/picked-case-study.njk

```diff
@@ -8,6 +8,10 @@
 <a class="feature-card" href="{{ pickedItem.url }}" data-treatment="bg-image">
   <span class="feature-card__eyebrow">Case study</span>
   <h3 class="feature-card__title">{{ pickedItem.data.title }}</h3>
+  {% if featureCard and featureCard.video %}
+  {% Img src=pickedItem.data.hero, alt="", width="570", height="330", class="feature-card__background", fetchpriority="high"%}
+  {% else %}
   {% Img src=pickedItem.data.hero, alt="", width="570", height="330", class="feature-card__background" %}
+  {% endif %}
```

Comment:
WDYT @philipwalton ? Is this complexity worth it? Or should we just add fetchpriority to this image regardless as it's an above the fold image on desktop anyway even when it's not LCP?

Reply:
Yeah, it's tricky. On mobile that image is never the LCP image, so I'd lean toward just turning off `loading=lazy` rather than setting `fetchpriority=high`.

Reply:
But that's the same with the first image in right hand column on mobile where we DO set fetchpriority="high" regardless.

Reply:
Given it will be just below the viewport, probably should remove `loading="lazy"` off the second image ANYWAY as it should never really be lazy loaded. Will add that to this PR.

But I'd vote for still explicitly adding `fetchpriority="high"` in cases where the above image is a video. The mobile LCP image still gets a high fetchpriority as well anyway and think the small hit to that on mobile (due to any bandwidth contention) is worth the bigger gain on desktop when this is LCP.